client/controllers.js

```diff
@@ -85,15 +124,147 @@ micControllers.controller('MainControl', ['$scope', '$location', 'Room', functio
 
 }]);
 
-micControllers.controller('AudienceControl', ['$scope', function($scope) {
+micControllers.controller('AudienceControl', ['$scope', '$sce', 'audienceRTC', '$rootScope', function($scope, $sce, audienceRTC, $rootScope) {
+  // Initialize micStatus with default settings of power = off (false) and the option to "Turn on your mic!"
+  // The power boolean is utilized to determine whether the views mic button will open a new peer connection with the presenter or close an existing connection.
+  // The command will toggle based on the power state so the user is aware what will happen.
+  console.log('all about the details ------------');
+  console.log($rootScope.details);
+
+  $scope.micStatus = {power: false, command: "Turn on your mic!"};
+
+  // only provide connect and disconnect functionality after ready (signal server is up, we have a media stream)
+  audienceRTC.ready(function () {
+    $scope.roomName = $rootScope.details.roomname.slice();
+    $scope.presenter = $rootScope.details.presenter.slice();
```

Comment:
Curious about why we're using $rootScope to get the room details instead of the room service...